suggestions/objects/suggestion.py

```diff
@@ -554,6 +571,7 @@ async def try_notify_user_of_decision(self, bot: SuggestionsBot):
             log.debug(
                 "User %s has dm messages disabled, failed to notify change to suggestion %s",
                 self.suggestion_author_id,
+                self.suggestion_author_id,
```

Comment:
Not sure if this was intentional, but I only see two variables in the string, but you have three passed through?